decouple the sope version number from the library interface number.

--- SOPE/NGCards/GNUmakefile.preamble.orig	Wed Oct 24 21:00:02 2012
+++ SOPE/NGCards/GNUmakefile.preamble	Fri Oct 26 12:11:38 2012
@@ -4,8 +4,8 @@ SOPE_ROOT=../..
 
 ADDITIONAL_CPPFLAGS += \
         -Wall -DCOMPILE_FOR_GSTEP_MAKE=1        \
-        -DSOPE_MAJOR_VERSION=$(MAJOR_VERSION)   \
-        -DSOPE_MINOR_VERSION=$(MINOR_VERSION)   \
+        -DSOPE_MAJOR_VERSION=$(SOPE_MAJOR_VERSION)   \
+        -DSOPE_MINOR_VERSION=$(SOPE_MINOR_VERSION)   \
         -DSOPE_SUBMINOR_VERSION=$(SUBMINOR_VERSION)
 
 ADDITIONAL_INCLUDE_DIRS += \
